api/app/controllers/venue_maps_controller.rb

```diff
@@ -46,6 +46,6 @@ def set_venue_map
     end
     # Only allow a list of trusted parameters through.
     def venue_map_params
-      params.permit(:group_id, :picture_name, :picture_path)
+      params.require(:venue_map).permit(:group_id, :picture_name, :picture_path)
```

Comment:
```suggestion
      params.permit(:group_id, :picture_name, :picture_path)
```
ユーザー画面からの新規登録ができなくなるから応急処置としてrequireは消した方いいかも
あとで何が正しい方法なのか模索してみる